### DISCUSSION TITLE
Why is there such a large fluctuation in pore water pressure

### DISCUSSION THREAD

**Discussion by wchrong** [Anura3D/Anura3D_OpenSource#168] [General]
When simulating landslides caused by rainfall infiltration, why does the pore water pressure change so dramatically without setting an initial groundwater level line? When setting a groundwater level line, why does the pore water pressure decrease linearly in the first stage? Do you know, can you tell me the reason? Thank you very much.
![2BRZZMUNEHAJ9O{HNO52XXM](https://github.com/Anura3D/Anura3D_OpenSource/assets/153343548/88ee5818-0659-4716-ad60-266bf95cca1e)

**Comment by murphy-john** [comment]
Large fluctuations in pressures and stresses is a common issue with dynamic explicit MPM code. It's caused by many things notably cell crossing errors.

**Comment by wchrong** [comment]
Thank you very much for your answer. Can you tell me some methods to reduce fluctuations? The changes in pore water pressure during the simulation process do not match what I measured. The simulated graph fluctuates too violently and there is no clear pattern.




苣漓
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Anura3D/Anura3D_OpenSource"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年12月14日(星期四) 凌晨0:05
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Anura3D/Anura3D_OpenSource] Why is there such a large fluctuation in pore water pressure (Discussion #168)





 
Large fluctuations in pressures and stresses is a common issue with dynamic explicit MPM code. It's caused by many things notably cell crossing errors.

**Comment by lucky3sunny** [comment]
I’m encountering the same problem as you. May I ask if you’ve found a solution?
